test/project/git/GitProjectTest.ts

```diff
@@ -197,4 +197,17 @@ describe("GitProject", () => {
             }).catch(done);
     }).timeout(5000);
 
+    it.skip("clones a project subdirectory", done => {
+        GitCommandGitProject.cloned(Creds, new GitHubRepoRef("pallets", "flask", "master",
+            GitHubDotComBase, "examples/flaskr"))
+            .then(gp => {
+                assert(!!gp.findFileSync("flaskr/__init__.py"));
+                gp.isClean()
+                    .then(r => {
+                        assert(r.success, "We should be able to get git status for a subdirectory");
```

Comment:
this comment confused me, I want it to say "in" or "from" or something instead of "for"